src/lib/select/_select-theme.scss

```diff
@@ -34,7 +34,7 @@
     }
   }
 
-  .md-select-content {
+  .md-select-panel {
```

Comment:
@crisbeto This will change the animation and take it off spec. Any other way to fix this?